hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/viewfs/ViewFileSystem.java

```diff
@@ -1130,16 +1132,49 @@ public BlockStoragePolicySpi getStoragePolicy(Path src) throws IOException {
    * Get the trash root directory for current user when the path
    * specified is deleted.
    *
+   * If CONFIG_VIEWFS_MOUNT_POINT_LOCAL_TRASH is not set, return
+   * the default trash root from targetFS.
+   *
+   * When CONFIG_VIEWFS_MOUNT_POINT_LOCAL_TRASH is set to true,
+   * 1) If path p is in fallback FS or from the same mount point as the default
+   *    trash root for targetFS, return the default trash root for targetFS.
+   * 2) else, return a trash root in the mounted targetFS
+   *    (/mntpoint/.Trash/<user>)
```

Comment:
The "<user>" causes javadoc to fail as an unknown tag. You need to quote using &lt; and &gt;.

Reply:
fixed. replaced with {}.